python/tvm/relay/frontend/tensorflow.py

```diff
@@ -637,10 +637,11 @@ def _impl(inputs, attr, params, mod):
         iou_threshold = np.atleast_1d(inputs[3].data.asnumpy())[0]
         # score_threshold was introduced from V3
         score_threshold = np.atleast_1d(inputs[4].data.asnumpy())[0] if len(inputs) > 4 else 0.0
+        pad_output = 'pad_to_max_output_size'
 
         # Generate data with shape (1, num_anchors, 5)
         scores = AttrCvt(op_name="expand_dims",
-                         ignores=['T_threshold'],
+                         ignores=['T_threshold', pad_output],
```

Comment:
We want to do padding if number of boxed is smaller than `max_output_size`?

Reply:
NMSv4 semantics allow for the user to define whether or not to pad the output using `pad_to_max_output_size=True/False`, see the example I provided in the test. Some pre-trained TF models use NMSv4 with pad_to_max_output_size=True, in which case, yes, they expect padding of the indices if the number of boxes is less than `max_output_size`, as well as an additional scalar output specifying the number of valid boxes.

Reply:
We need to handle ```True``` case instead of just ignoring it?

Reply:
It's not ignored, this PR adds explicit support to `__nms()` for the `pad_to_max_output_size=True` case, see lines 671/2.

Reply:
Got it. LGTM.